public/class-wa-external-header-v2-public.php

```diff
@@ -90,39 +90,40 @@ public function __construct($plugin_name, $version, $options_group_name)
 
     public function wp_head()
     {
-            $content_unit_category = $this->user_config['content_unit_category'];
-            $tns_path = $this->user_config['tns_tracking_path'];
-
-            if ($tns_path) {
-                //ex <meta content="MMK/COSTUME/ExBlogger" name="tns-path" />
-                $this->head = preg_replace(
-                    "/<meta.*?name=\"tns-path\"[\s\/]+>/",
-                    "<meta content=\"$tns_path\" name=\"tns-path\">",
-                    $this->head
-                );
-
-                /*//ex <div data-tns-path="costume/external">
-                $this->header = preg_replace(
-                    "/<div data-tns-path=\"(.*?)\">/",
-                    "<div data-tns-path=\"$tns_path\">",
-                    $this->header
-                );*/
-            }
-
-            echo "
-                <meta name=\"banner-category\" content=\"$content_unit_category\">
-
+        $content_unit_category = $this->user_config['content_unit_category'];
+        $tns_path = $this->user_config['tns_tracking_path'];
+        $hideShell = isset($this->user_config['bp_hide_shell']);
```

Comment:
Hvis bp_hide_shell er false, vil $hideShell stadig blive true.

$hideShell = isset($this->user_config['bp_hide_shell']) ? $this->user_config['bp_hide_shell'] : 'false';